src/labels/CustomLabels.labels

```diff
@@ -1872,6 +1872,14 @@ If you&apos;re not sure how to resolve these errors, post a message in the Nonpr
         <shortDescription>Message displayed indicating data is not available when enhanced RDs are enabled</shortDescription>
         <value>This page is not available in Enhanced Recurring Donations. Refresh your browser and return to NPSP Settings.</value>
     </labels>
+    <labels>
+        <fullName>RD2_ScheduleInstallmentIsRequired</fullName>
+        <categories>Recurring-Donations, Error</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Every installment Opportunity should have matching RD Schedule installment</shortDescription>
+        <value>The Opportunity cannot be matched with the Recurring Donation Schedule installment.</value>
```

Comment:
@andrewyu-salesforce - What conditions would generate this error? Is this something that a user would see?

Reply:
This is part of PR #4903 . If this happens, the most probable cause is a programatic error (edge scenario or something not already considered).